modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/dashboard/hocs/InterestsCard.tsx

```diff
@@ -71,7 +72,11 @@ const InterestsCard: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 	];
 
 	return (
-		<Card className='interests-card-root' minHeight={536}>
+		<Card
+			className='interests-card-root'
+			id={Containers.TopInterestsAsOfYesterday}
```

Comment:
Rename it to TopInterestsAsOfYesterdayCard

Reply:
done